lib/util/platform.js

```diff
@@ -707,14 +707,11 @@ shaka.util.Platform = class {
   /**
    * Returns true if MediaKeys is polyfilled
```

Comment:
I would update the JSDoc now that the polyfillType is non-optional.

> Returns true if MediaKeys is polyfilled by the specified polyfill.

Reply:
Done!